### PR TITLE
Add more stats for httpd, query executor

### PR DESCRIPTION
**Pull request by mark-rushakoff**
Four new stats in httpd:

* `queryReqDurationNs`: time spent inside GET /query
* `writeReqDurationNs`: time spent inside POST /write
* `reqDurationNs`: time spent handling any HTTP request
* `reqActive`: number of currently active HTTP requests

Two new stats in query executor:

* `queryDurationNs`: time spent executing queries
* `queriesActive`: number of currently active/executing queries

### CODE REVIEW

File: tsdb/query_executor.go [hunk 1]
```diff
@@ -140,7 +151,12 @@ func (q *QueryExecutor) ExecuteQuery(query *influxql.Query, database string, chu
 	results := make(chan *influxql.Result)
 
 	go func() {
-		defer close(results)
+		q.statMap.Add(statQueriesActive, 1)
+		defer func(start time.Time) {
+			close(results)
```
Comment (e-dard) [inline tsdb/query_executor.go]: Not sure why the `close` has moved into here? I think the following would be fine:

```go
defer close(results)
defer func(start time.Time) { ... }(time.Now())
```

**Comment by e-dard** [comment]
@mark-rushakoff Ben is currently moving the query executor to the `influxql` package in #5663. Once that's merged this'll need a rebase.

**Comment by mark-rushakoff** [comment]
@e-dard Rebased and comment addressed.

**Comment by e-dard** [comment]
LGTM :+1: /cc @jwilder

**Comment by jwilder** [comment]
:+1:

**Comment by jwilder** [comment]
@mark-rushakoff Could you add this to the changelog too?

**Comment by mark-rushakoff** [comment]
I'll update the changelog, wait for a green build, and merge.

**Pull request merged**